mobile/lib/constants/config.dart

```diff
@@ -78,7 +78,8 @@ class Config {
 
   static String get sentryDsn => dotenv.env['SENTRY_DSN'] ?? '';
 
-  static String get termsUrl => dotenv.env['TERMS_URL'] ?? '';
+  static String get termsUrl =>
+      'https://staging-docs.airqo.net/#/mobile_app/privacy_policy';
```

Comment:
Updated link to privacy policy. cc @123MwanjeMike

Reply:
Hi @123MwanjeMike , I hope you are on top of this?

Also, you can proceed with the domain mapping requests and be ready to address any follow up tasks that will arise because of this.

Reply:
Thanks @NoahNsimbe
Could we change `staging-docs.airqo.net` to `docs.airqo.net` or pass it as an environment variable

Reply:
Thanks @123MwanjeMike 
Let me change it to `docs.airqo.net`. Passing it as an environment variable won't make any difference since it is a public link and we don't have control over environment variables once me make a release.

We can explore remote configuration but it has limitations.